gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/E2EMappingIT.java

```diff
@@ -207,7 +211,7 @@ private String parseMessageToString(InboundMessage inboundMessage) {
 
     private boolean isEhrExtractTranslated() {
```

Comment:
this method, isEhrExtractTranslated() is repeated and the other method that does the same is called "isEhrMigrationCompleted". one of the methods should be picked and the other method should be deleted

Reply:
Filename issue is reversed so we know the process is correct.